jazz_environments/components/utils.js

```diff
@@ -48,7 +48,7 @@ var getEnvironmentDatabaseKeyName = function(key) {
 var getSchemaKeyName = function(key) {
     // Convert database key name back, as per schema
 
-    if (key === undefined || key === null) {
+    if (key === undefined || !key) {
```

Comment:
don't need a check for undefined since you are already checking for `!key`

Reply:
Done